app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -186,5 +147,31 @@ def experience_to_hash(experience)
         description: experience.details,
       }
     end
+
+    def qualifications
+      {
+        gcses: application_form.application_qualifications.gcses.map do |gcse| qualification_to_hash(gcse) end,
+        degrees: application_form.application_qualifications.degrees.map do |degree| qualification_to_hash(degree) end,
+        other_qualifications: application_form.application_qualifications.other.map do |other| qualification_to_hash(other) end,
+      }
+    end
+
+    def qualification_to_hash(qualification)
+      {
+        qualification_type: qualification.qualification_type,
+        subject: qualification.subject,
+        grade: "#{qualification.grade}#{' (Predicted)' if qualification.predicted_grade}",
```

Comment:
should this evaluate to `nil` if `qualification_grade` is `nil` (I think it will evaluate to empty string)? Would make it more consistent with other fields like `institution_details`

Reply:
Do we expect `qualification_grade` to be nil when the application is viewable to providers?